project/src/main/java/com/google/sps/data/DataAccessor.java

```diff
@@ -68,7 +67,7 @@ public boolean isRegistered(String userId) {
   * @param userId the id of the user that we want to register to our system
   */
   public void registerUser(String userId) {
-    checkArgument(!Strings.isNullOrEmpty(userId), "Invalid user ID");
+    ValidationUtils.validateNonEmptyString(userId, "Invalid user ID.");
```

Comment:
This is a redundant change: the method has the exact same line, calling the method or having the checkArgument in place isn't adding anything new.

Note that the reason to have a Validations class is for specific validations that have logic in them (as there's already a Strings.isNullOrEmpty, you may simply use it:)).

Reply:
done